frontend/src/pages/PipelineDetailsV1.tsx

```diff
@@ -195,8 +204,18 @@ const PipelineDetailsV1: React.FC<PipelineDetailsV1Props> = ({
                     )}
                     <div className={css.summaryKey}>Uploaded on</div>
                     <div>{formatDateString(pipeline.created_at)}</div>
-                    <div className={css.summaryKey}>Description</div>
-                    <Description description={pipeline.description || ''} />
+                    {pipeline.description ? (
+                      <React.Fragment>
+                        <div className={css.summaryKey}>Pipeline Description</div>
+                        <Description description={pipeline.description || ''} />
+                      </React.Fragment>
+                    ) : null}
+                    {createVersionDescription() ? (
+                      <React.Fragment>
+                        <div className={css.summaryKey}>Version Description</div>
+                        <Description description={createVersionDescription() || ''} />
+                      </React.Fragment>
+                    ) : null}
```

Comment:
Why do we show both pipeline description and version description? Can we show only the version createVersionDescription() if this is available, and show pipeline description if createVersionDescription() not exist?

Reply:
Sure. I wanted to show both just in case it was useful to have both. That said, it may be too much. 

So happy with either.

Reply:
Right, the reason I want to simplify the summary card is that user usually saw the description already when they are on `Pipeline List` page. That said, I think your point also makes sense where people want to see both the pipeline description and the version description. I tend to move the pipeline description to a `pipeline template detail` tab (such tab doesn't exist today), so it can balance the weight of information we show in summary card. You don't need to make that change in this PR.

Something you can do is to show the version description in `Pipeline List` page too: 

![pipelinelistversiondescription](https://user-images.githubusercontent.com/37026441/130270994-c47f47ec-e3f8-4f45-8de1-ba68a3fb29ad.png)

You can do it in a separate PR as well.

Reply:
Added in #6405